main.go

```diff
@@ -15,6 +17,15 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	el, err := NewEventsLogger("crawl-events.json")
+	if err != nil {
+		panic(err)
+	}
+
+	r, w := io.Pipe()
```

Comment:
this should probably be moved inside the logger i think

Reply:
moved the initialization of event logging inside `NewLogger`; let's keep main simple and stupid :)